kvmagent/kvmagent/test/test_vm_artifact.py

```
@@ -0,0 +1,132 @@
+        self.old_model_root = vm_artifact.MODEL_CENTER_ROOT
+        self.old_view_root = vm_artifact.VM_VIEW_ROOT
+        vm_artifact.MODEL_CENTER_ROOT = self.model_root
+        vm_artifact.VM_VIEW_ROOT = self.view_root
+
+    def tearDown(self):
+        vm_artifact.MODEL_CENTER_ROOT = self.old_model_root
+        vm_artifact.VM_VIEW_ROOT = self.old_view_root
+        shutil.rmtree(self.tmp, ignore_errors=True)
+
+    def test_path_escape_is_rejected(self):
+        with self.assertRaises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)